suite-common/formatters/src/formatters/prepareCryptoAmountFormatter.ts

```diff
@@ -1,28 +1,31 @@
 import BigNumber from 'bignumber.js';
+import { G } from '@mobily/ts-belt';
 
 import { networksCompatibility as NETWORKS, NetworkSymbol } from '@suite-common/wallet-config';
 import {
     localizeNumber,
     networkAmountToSatoshi,
     formatCoinBalance,
+    formatNetworkAmount,
 } from '@suite-common/wallet-utils';
 import { PROTO } from '@trezor/connect';
 
 import { makeFormatter } from '../makeFormatter';
 import { FormatterConfig } from '../types';
-import { prepareCurrencySymbolFormatter } from './prepareCurrencySymbolFormatter';
+import { prepareNetworkSymbolFormatter } from './prepareNetworkSymbolFormatter';
 
 export type CryptoAmountFormatterInputValue = string | number | BigNumber;
 
 export type CryptoAmountFormatterDataContext = {
     isBalance?: boolean;
     symbol?: NetworkSymbol;
     withSymbol?: boolean;
+    isSatoshis?: boolean;
 };
 
 export const prepareCryptoAmountFormatter = (config: FormatterConfig) =>
     makeFormatter<CryptoAmountFormatterInputValue, string, CryptoAmountFormatterDataContext>(
-        (value, { symbol, isBalance, withSymbol = true }) => {
+        (value, { symbol, isBalance, withSymbol = true, isSatoshis = false }) => {
```

Comment:
Uff this is going crazy :D Isn't `isBalance` basically reversed `isSatoshis`, because if it's `balance=true` it means that `isSatoshis` must be `false`. If it's amount like from transaction is not balance so it's `balance=false` then it must be `isSatoshis=true`.

It looks that `isSatoshis` is only duplicity for isBalance.

Reply:
https://github.com/trezor/trezor-suite/pull/7401/commits/95ca5687e89f5cc78ddc61c76d75edb276e418b0